### PR TITLE
Widgets: Marquee fixes for marquee up/down

### PR DESCRIPTION
**Pull request by rubenpingol-xibo**
relates to xibosignageltd/xibo-private#438

### CODE REVIEW

File: modules/src/xibo-elements-render.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ jQuery.fn.extend({
         ((options.effect === 'marqueeLeft') ? 'left' : 'right');
 
       // Make sure the speed is something sensible
-      options.speed = (options.speed === 0) ? 1 : options.speed;
+      options.speed = (options.speed === 0) ? 1 : options.speed / 25 * 1000;
```
Comment (dasgarner) [inline modules/src/xibo-elements-render.js:139]: What is the reason for dividing by 25, and can be add a comment explaining that?

Comment (rubenpingol-xibo) [reply modules/src/xibo-elements-render.js:139]: This calculation gives us a result of `80` as a value for `options.speed`. This value is the speed we are already using from `xibo-text-render.js` which is I believe based on the **85 milliseconds** default scrollamount as pixels per second.

**Review by maurofmferrao — APPROVED** [review]

File: modules/src/xibo-elements-render.js (hunk 1 above)
Comment (dasgarner) [reply modules/src/xibo-elements-render.js:140]: Understood! I think a comment explaining this for "future Ruben" would be awesome, but you can merge this for testing in the meantime.

**Review by dasgarner — APPROVED** [review]

File: modules/src/xibo-elements-render.js (hunk 1 above)
Comment (rubenpingol-xibo) [reply modules/src/xibo-elements-render.js:140]: Yep, got it @dasgarner. I'll take this into account. Thanks!

**Review by maurofmferrao — APPROVED** [review]
Looks good to me!

**Pull request merged**